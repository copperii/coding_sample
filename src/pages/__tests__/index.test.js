import React from 'react'
// import { render } from '@testing-library/react'
import { create } from 'react-test-renderer'
// import Pages from '../index'

const Button = () => {
  return <button>dummy button</button>
}

describe('dummy component for test page', () => {
  test('Matches the snapshot', () => {
    const button = create(<Button />)
    expect(button.toJSON()).toMatchSnapshot()
  })
})
