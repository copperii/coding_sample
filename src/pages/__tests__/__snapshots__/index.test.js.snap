// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dummy component for test page Matches the snapshot 1`] = `
<button>
  dummy button
</button>
`;
